src/Interpreter.test.ts

```diff
@@ -1465,6 +1465,12 @@ test('interpret two-column template', async () => {
     )
     expect(votes).toMatchInlineSnapshot(`
       Object {
+        "750000015": Array [
+          "yes",
+        ],
+        "750000016": Array [
+          "yes",
+        ],
```

Comment:
These are in [the scanned ballot](https://github.com/votingworks/hmpb-interpreter/blob/main/test/fixtures/choctaw-county-mock-general-election-choctaw-2020-e87f23ca2c/filled-in-p2.png?raw=true) but weren't here before.